### PR TITLE
adding Gregor Martynus

### PR DESCRIPTION
**Pull request by gr2m**
okay to add flights? Might be nice to coordinate with others? But happy to remove them if you prefer

### CODE REVIEW

**Comment by jessfraz** [comment]
yeah for sure!

**Comment by jamiebuilds** [comment]
you deleted me

**Comment by gr2m** [comment]
@thejameskyle nooooooooo sorry racing condition! I used GitHub.com’s UI for the PR, I must have missed it so sorry!

**Comment by jessfraz** [comment]
ugh i hate git sometimes

File: wontfix_cabal/participants.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ sending a pull request!
 - Jess Frazelle (organizer, [@jessfraz on GitHub](https://github.com/jessfraz), [@jessfraz on Twitter](https://twitter.com/jessfraz))
 - Katrina Owen (organizer, [@kytrinyx on GitHub](https://github.com/kytrinyx),
   [@kytrinyx on Twitter](https://twitter.com/kytrinyx))
-- James Kyle (idiot, [@thejameskyle on GitHub](https://github.com/thejameskyle), [@thejameskyle on Twitter](https://twitter.com/thejamekyle))
```
Comment (gr2m) [inline wontfix_cabal/participants.md:9]: @thejameskyle btw your twitter url misses an s I think?

Comment (jessfraz) [reply wontfix_cabal/participants.md:9]: oh ill fix that too but I added him back :)

Comment (jamiebuilds) [reply wontfix_cabal/participants.md:9]: tru

**Pull request merged**